modules/router/templates/router_include.conf.erb

```diff
@@ -51,7 +51,9 @@ location ~ ^/performance/ {
   proxy_pass http://varnish;
 }
 
-location ~ ^/contact/govuk/(service-feedback|problem_reports)$ {
+# This list of URLS should be kept in sync with the POST-able routes in the
```

Comment:
I think that's the best we can do at the moment as nginx doesn't really support env vars in a nice way and that'd be the best thing IMO.